esp-hal/src/i2c/rtc.rs

```diff
@@ -0,0 +1,737 @@
+use core::time::Duration;
+
+use crate::{
+    gpio::{InputPin, OutputPin, RtcFunction, RtcPin},
+    peripherals::RTC_I2C,
+};
+
+const RC_FAST_CLK: u32 = property!("soc.rc_fast_clk_default");
+
+#[doc(hidden)]
+pub trait Sda: RtcPin + OutputPin + InputPin {
```

Comment:
These shouldn't be hidden, the documentation should show the user which pins implement these traits.

Reply:
Ah, `OtgFs` does the same thing, so I followed suit. I'll change this one at least.